MCprep_addon/materials/prep.py

```diff
@@ -305,9 +306,10 @@ def execute(self, context):
 				"Nothing modified, be sure you selected objects with existing materials!"
 			)
 
-		addon_prefs = util.get_user_preferences(context)
 		self.track_param = context.scene.render.engine
-		self.track_exporter = addon_prefs.MCprep_exporter_type
+		# TODO: Rework exporter tracking
+		# addon_prefs = util.get_user_preferences(context)
+		# self.track_exporter = addon_prefs.MCprep_exporter_type
```

Comment:
I think the tracker is still worthwhile, as it helps us understand the ecosystem of what's being used and how to best support everyone.

Main question is: do we have the header auto assign a value of e.g. jmc2obj to the exporter type, or should we add two more types which are  jmc2obj_cmobj (or something like that) and the same for Mineways? I'm a bit torn. It could be quite nice to know how well adopted the common format is, to know what feature work and whether it applies to a good % of the MCprep userbase.

Reply:
I think we could use the output of `get_exporter` (something like `get_exporter(context).value` to retrive the underlying string from the enum, assuming it's not `None`) as it would be a 1-1 mapping, although we'd then have to concert the Mineways and jmc2OBJ references in the tracker to `mineways-c` (Mineways before CommonMCOBJ) and `jmc2obj-c` (jmc2OBJ before CommonMCOBJ). It would also allow tracking the use of unknown exporters as well

Reply:
Thinking abut this further, I think we should use the string in the `exporter` key of the header. If we use the enum, we won't be able to know if a new OBJ exporter is gaining traction (and thus could be better supported in MCprep), but the exporter key would make it more clear what specific exporters are being used

Reply:
Remind me, what would be the values we potentially get in the exporter? that's the mineways-defined or jmc2obj-defined names for themselves, right?

I think having something like jmc2obj_cmobj would be fantastic or at least _somehow__ knowing which imports comes form the commonmcobj spec in the analytics data. A canned suffix added to the `exporter` string could do. Knowing the depth of adoption of the new standard helps understand how many people can make use of a tool we build that depends on that (if e.g. less than 10% of obj imports are using the spec, that could be good rationale to not yet spend ages working on the bridge tool just yet for instance).

Reply:
> Remind me, what would be the values we potentially get in the exporter? that's the mineways-defined or jmc2obj-defined names for themselves, right?

It would be the name of the exporter, so `jmc2obj`, `mineways`, `helpicantexitvim`, etc.